RecastDemo/Source/ConvexVolumeTool.cpp

```diff
@@ -130,14 +130,18 @@ void ConvexVolumeTool::handleMenu()
 
 	imguiLabel("Area Type");
 	imguiIndent();
-	if (imguiCheck("Grass", m_areaType == SAMPLE_POLYAREA_GRASS))
-		m_areaType = SAMPLE_POLYAREA_GRASS;
-	if (imguiCheck("Road", m_areaType == SAMPLE_POLYAREA_ROAD))
-		m_areaType = SAMPLE_POLYAREA_ROAD;
+// 	if (imguiCheck("Ground", m_areaType == SAMPLE_POLYAREA_GROUND))
+// 		m_areaType = SAMPLE_POLYAREA_GROUND;
```

Comment:
As noted above
> updated the imguiCheck for convex volumes so that all SamplePolyAreas values are covered, except GROUND, which corresponds to zero, i.e. the default area value. Maybe adding GROUND could be useful too. To be discussed.

More precisely, the `SAMPLE_POLYAREA_GROUND` area flag value is zero. Creating a convex volume with `SAMPLE_POLYAREA_GROUND` area actually makes holes in the built mesh, which is not coherent with the label "Ground".
Either we keep "Ground" and give it another non-zero value, or rename it "Hole", or "No mesh" or something alike ?